orchestrator/nodes/dockerComposeRemoteNode/index.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+var RemoteNode = require("../remoteNode/index").Handler;
+
+// This should be external - but since it's bugged....
+var docker = require('../../docker/harbor-master');
+
+function makeId(length) {
+  var text = "";
+  var possible = "abcdef0123456789";
+
+  for (var i = 0; i < length; i++) {
+    text += possible.charAt(Math.floor(Math.random() * possible.length));
+  }
+  return text;
+}
+
+
+class DataHandler extends RemoteNode {
+  constructor(image, id) {
+    super();
+    this.info = {
+      userid: id,
+      image: image,
+      enabled: false
+    };
+
+    // TODO: socket path should be configurable
```

Comment:
Just add an environment variable for this part with a default value. No reason to delay this task

Reply:
Ok